builtin/providers/aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -44,6 +46,20 @@ func trustedSignersConf() []interface{} {
 	return []interface{}{"1234567890EX", "1234567891EX"}
 }
 
+func lambdaFunctionAssociationsConf() *schema.Set {
+	s := []interface{}{
+		map[string]interface{}{
+			"event_type": "viewer-request",
+			"lambda_arn": "arn:aws:lambda:us-east-1:999999999:function1:alias",
+		},
+		map[string]interface{}{
+			"event_type": "origin-response",
+			"lambda_arn": "arn:aws:lambda:us-east-1:999999999:function2:alias",
+		},
```

Comment:
Oh, I see, sorry... docs I love need to are for the collection. Clicking through to the [LambdaFunctionAssociation](http://docs.aws.amazon.com/AmazonCloudFront/latest/APIReference/API_LambdaFunctionAssociation.html) model, it defines EventType and LambdaFunctionArn